config/deploy.rb

```diff
@@ -58,6 +59,7 @@
     end
   end
 
+  #TODO FIX: remove password from wp core install command
   desc "Wp Core Install"
```

Comment:
I wouldn't expect this to be a cap task done with each release but rather a one-time thing done on initial setup. If this is the case, it would make more sense in the ansible playbook.

Have you tested multiple deploys to the same server?